backend/app/src/main/kotlin/io/tolgee/configuration/PortAvailability.kt

```diff
@@ -0,0 +1,32 @@
+package io.tolgee.configuration
+
+import okhttp3.internal.closeQuietly
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.web.servlet.ServletContextInitializer
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import java.io.IOException
+import java.net.ServerSocket
+import kotlin.system.exitProcess
+
+@Configuration
+class PortAvailability {
+
+  private val logger = LoggerFactory.getLogger(PortAvailability::class.java)
+
+  @Bean
+  fun portAvailabilityChecker(
+    @Value("\${server.port:8080}") port : Int
```

Comment:
It might be better to use these annotation and also check the management port availability.

https://stackoverflow.com/a/38894869

Reply:
`@LocalServerPort` looks like it's meant to be used only for tests.

I'll check the management port.

Reply:
Oh, OK.